doc/Known_issues.md

```diff
@@ -77,3 +81,42 @@ environment was not completely restored
 Unknown. Make sure that the {renv} environment is completely updated by running
 `renv::status()`. Try to restore the environment running
 `renv::restore(confirm = FALSE)` if the environment is not completely restored.
+
+### Problem
+
+When the learner has had a GitHub account for long, and not used much, GitHub
+may ask to use an "access token" to confirm their identity.
+
+When trying to clone their own private repo then, instead of being prompted by
+Rstudio to authenticate with the browser, an authentication issues rises.
+
+### Cause
+
+GitHub in July 2020, announced their intent to require the use of token-based
+authentication for all authenticated Git operations. Beginning August 13, 2021,
+they no longer accept account passwords when authenticating Git operations on
+GitHub.com.
+
+It affects the following workflows: - Command line Git access - Desktop
+applications using Git (GitHub Desktop is unaffected) - Any apps/services that
+access Git repositories on GitHub.com directly using their password
+
+### Solution
+
+Learners should use a personal access token
+(<https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens>)
+
+The easiest solution is to create a perosnal access token (classic):
```

Comment:
Suggestion: Collapse lines 106-109 into one (they are redundant), and link the text "create a personal access token" instead of adding the link in parentheses afterwards.